### PR TITLE
Add E-Label

### PR DESCRIPTION
**Pull request by filipecarneiro**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/e-label.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - .NET
+  - C#
+  - Docker
+  - K8S
+tags:
+  - Miscellaneous
+  - Community-Supported Agriculture (CSA)
+  - Inventory Management
+  - URL Shorteners
+depends_3rdparty: false
```
Comment (kokomo123) [inline software/e-label.yml:17]:
```suggestion
```
No need for this line as it's already default to false.

File: software/e-label.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: "E-Label"
+website_url: "https://filipecarneiro.github.io/ELabel/"
+source_code_url: "https://github.com/filipecarneiro/ELabel"
+description: "Open-source solution for electronic labels, with QR Codes, on wine bottles sold within the European Union. This initiative seeks to enhance transparency, efficiency, and information accessibility in the wine industry."
```
Comment (Rabenherz112) [inline software/e-label.yml:4]: The description should only describe what the software does. Additionally terms such as open-source should be avoided ([see here](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines))
Possible alternative description; however, feel free to re-write it yourself according to your preferences!
```suggestion
description: "Solution for electronic labels, with QR code support on wine bottles sold within the European Union."
```

File: software/e-label.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+licenses:
+  - MIT
+platforms:
+  - .NET
+  - C#
+  - Docker
+  - K8S
```
Comment (Rabenherz112) [inline software/e-label.yml:11]: The deployment sections only mention docker as a way to install this software.
```suggestion
  - Docker
```

File: software/e-label.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+tags:
+  - Miscellaneous
+  - Community-Supported Agriculture (CSA)
+  - Inventory Management
+  - URL Shorteners
```
Comment (Rabenherz112) [inline software/e-label.yml:16]: Miscellaneous and URL Shorteners don't seem to fit here.
If your software does a built-in Inventory Management tool, I am fine with this tag, but I couldn't find anything on the GitHub project mentioning it.
```suggestion
  - Community-Supported Agriculture (CSA)
```

File: software/e-label.yml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+demo_url: "https://qr.doutorpc.net/l/GR904-2015"
```
Comment (Rabenherz112) [inline software/e-label.yml:17]: > Demo links should only be used for interactive demos

I wouldn't count this static website as interactive, but maybe other reviews want to share their opinion on that.
```suggestion
```

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]

**Comment by nodiscc** [comment]
@filipecarneiro please address the points mentioned in the review

**Comment by nodiscc** [comment]
@filipecarneiro ping

**Review by kokomo123 — APPROVED** [review]
Looking clean and concise now, think we can merge this, thanks nodiscc 💯

**Review by nodiscc — APPROVED** [review]

**Pull request merged**